### PR TITLE
Reproduced results - experiments-msmacro-passage

### PR DESCRIPTION
**Pull request by Seun-Ajayi**
Everything worked!

System Information:

Operating System: Ubuntu 20.04.6 LTS on GCP
Number of CPUs: 8
Amount of Memory: 32GB
Disk Space: 100GB
Environment Configuration:

Python Version: Python 3.10.13
Java Version:
openjdk 11.0.13 2021-10-19
OpenJDK Runtime Environment JBR-11.0.13.7-1751.21-jcef (build 11.0.13+7-b1751.21)
OpenJDK 64-Bit Server VM JBR-11.0.13.7-1751.21-jcef (build 11.0.13+7-b1751.21, mixed mode)

### CODE REVIEW

**Comment by lintool** [comment]
Please continue along the onboarding path until you reach the end of the exercises, and then we'll take a look all at once.

**Comment by lintool** [comment]
Why have the queries and qrels been deleted?

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -333,4 +333,4 @@ Before you move on, however, add an entry in the "Reproduction Log" at the botto
 + Results reproduced by [@Melissa1412](https://github.com/Melissa1412) on 2023-11-05 (commit [`acd969f`](https://github.com/castorini/pyserini/commit/acd969f8f234126c272d70d55d047a3804b52ff8))
 + Results reproduced by [@aliranjbari](https://github.com/aliranjbari) on 2023-11-08 (commit [`12cbb11`](https://github.com/castorini/pyserini/commit/12cbb11efbf1d82c2be84bc376e1aceffcaced31))
 + Results reproduced by [@salinaria](https://github.com/salinaria) on 2023-11-11 (commit [`086e16b`](https://github.com/castorini/pyserini/commit/086e16be28b7dc6022f8582dbd803824dc2c1ad2))
-+ Results reproduced by [@oscarbelda86](https://github.com/oscarbelda86) on 2023-11-13 (commit [`086e16b`](https://github.com/castorini/pyserini/commit/086e16be28b7dc6022f8582dbd803824dc2c1ad2))
\ No newline at end of file
++ Results reproduced by [@oscarbelda86](https://github.com/oscarbelda86) on 2023-11-13 (commit [`086e16b`](https://github.com/castorini/pyserini/commit/086e16be28b7dc6022f8582dbd803824dc2c1ad2))+ Results reproduced by [@Seun-Ajayi](https://github.com/Seun-Ajayi) on 2023-11-13 (commit [`086e16b`](https://github.com/castorini/pyserini/commit/086e16be28b7dc6022f8582dbd803824dc2c1ad2))
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:336]: Your entry should be on it's own line...

Comment (Seun-Ajayi) [reply docs/experiments-msmarco-passage.md]: Fixed

Comment (Seun-Ajayi) [reply docs/experiments-msmarco-passage.md]: And I want to ask, which experiment am I meant to take next? This information was not indicated in the last exercise I did, the `conceptual-framework2`

**Review by lintool — APPROVED** [review]

**Pull request merged**